app/neo/Overview.tsx

```diff
@@ -21,6 +22,40 @@ type OverviewProps = {
   control?: ReactNode;
 };
 
+export const CreateNewArtefactButton = ({ title, open }: { title: string; open: () => void }) => {
+  const { addElement, importFromFile, importFromMarket } = useKnownHotkeys(title);
+  useHotkeys(addElement.hotkey, open, { keydown: false, keyup: true, scopes: ['neo'] });
+  const addTooltip = title.match('Import Projects') ? importFromFile.label + '\n' + importFromMarket.label : addElement.label;
```

Comment:
I think this was like this, because we had one care (on the project overview) which had two functions. However this are now two buttons, So this should no longer be needed.
This would mean that we could add the hotkey as parameter (e.g. addElement).